.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  xcode-build:
+    name: Xcode Build
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        platform: ['iOS_13', 'iOS_12']
+      fail-fast: false
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Bundle Install
+        run: bundle install --gemfile=Example/Gemfile
+      - name: Select Xcode Version (11.3.1)
+        run: sudo xcode-select --switch /Applications/Xcode_11.3.1.app/Contents/Developer
+        if: matrix.platform == 'iOS_13'
+      - name: Select Xcode Version (10.3)
+        run: sudo xcode-select --switch /Applications/Xcode_10.3.app/Contents/Developer
+        if: matrix.platform == 'iOS_12'
```

Comment:
This is definitely not the ideal way to set this up... but I can't really figure out a better way to do this sort of mapping in the config. And it works, so 🤷🏻‍♂️